cmake/compilers/MSVC.cmake

```diff
@@ -76,5 +76,9 @@ if (TBB_FILE_TRIM)
         "$<$<COMPILE_LANGUAGE:CXX>:/d1trimfile:${NATIVE_TBB_PROJECT_ROOT_DIR}\\>"
         "$<$<COMPILE_LANGUAGE:CXX>:/d1trimfile:${CMAKE_SOURCE_DIR}/>")
 endif()
+if (TBB_RELEASE_PDB AND CMAKE_BUILD_TYPE MATCHES "Release")
+    set(TBB_LIB_COMPILE_FLAGS ${TBB_LIB_COMPILE_FLAGS} /Zi)
+    set(TBB_LIB_LINK_FLAGS ${TBB_LIB_LINK_FLAGS} /DEBUG /OPT:REF /OPT:ICF)
+endif()
```

Comment:
Isn't it essentially the same as RelWithDebInfo? Meaning you'll have same /Zi and /DEBUG flags there and .pdb files as well.

Reply:
RelWithDebInfo also changes its default for inlined functions from /Ob2 in Release to /Ob1 in RelWithDebInfo

Not sure, how it affects TBB performance.